src/main/java/uk/gov/hmcts/reform/bsp/common/service/transformation/BulkScanFormTransformer.java

```diff
@@ -49,7 +52,14 @@
         Map<String, String> ocrToCCDMapping = getOcrToCCDMapping();
 
         return ocrDataFields.stream()
+            .filter(ocrDataField -> Objects.nonNull(ocrDataField.getName()))
+            .filter(ocrDataField -> Objects.nonNull(ocrDataField.getValue()))
             .filter(ocrDataField -> ocrToCCDMapping.containsKey(ocrDataField.getName()))
+            .peek(ocrDataField -> {
+                if (ocrDataField.getValue() == null) {
```

Comment:
having known we have code in line 56, why do we need this if?

Reply:
Absolutely - will remove once we get to bottom of this SIT issue - just want to be very verbose for now

Reply:
ok